xml/System.Text.Json/JsonDocument.xml

```diff
@@ -65,11 +65,13 @@
         <Parameter Name="readerOptions" Type="System.Text.Json.JsonReaderOptions" />
       </Parameters>
       <Docs>
-        <param name="utf8Json">To be added.</param>
-        <param name="readerOptions">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="utf8Json">JSON text to parse.</param>
+        <param name="readerOptions">Options to control the reader behavior during parsing.</param>
+        <summary>Parse a sequence as UTF-8-encoded text representing a single JSON value into a JsonDocument.</summary>
+        <returns>A JsonDocument representation of the JSON value.</returns>
         <remarks>To be added.</remarks>
```

Comment:
The remarks for the parse methods are important, in particular the memory idempotentcy requirement on the caller.

Reply:
Interesting. This is the first time I notice remarks in the triple slash xml files inside corefx/artifacts/bin. That would be a new feature I should add to my tool.  I'll update the PR in a few minutes. Thanks for noticing, @bartonjs.

Reply:
Oooh I also noticed you added <typeparamref> elements. I should port those too.